python/tk_multi_workfiles/browser_form.py

```diff
@@ -669,3 +679,37 @@ def _on_step_filter_changed(self, step_list):
         :param step_list: A list of Shotgun Step dictionaries.
         """
         self.step_filter_changed.emit(get_filter_from_filter_list(step_list))
+
+    # Task status filter methods
+    def populate_task_status_list(self):
+        """
+        Populate the task status list from the project schema.
+        """
+        app = sgtk.platform.current_bundle()
+        project_id = app.context.project.get("id")
+        if project_id:
+            logger.debug(
+                f"Task Status Filter: getting statuses from project {project_id}"
+            )
+            task_status_list = app.shotgun._sg.schema_field_read(
```

Comment:
You should not need to access the `_sg` property, just use `shotgun` itself
```suggestion
            task_status_list = app.shotgun.schema_field_read(
```

Reply:
Didn't work to me. 

I see the [shotgun](https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/master/app.py#L166) property returns a `DebugWrapperShotgun`. From there I can access the `_sg` [instance](https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/master/app.py#L189) as property from it.